lbrynet/cryptstream/CryptBlob.py

```diff
@@ -91,7 +91,7 @@ def close(self):
 
     def _write_buffer(self):
         num_bytes_to_encrypt = (len(self.buff) // AES.block_size) * AES.block_size
-        data_to_encrypt,  self.buff = self.buff[:num_bytes_to_encrypt], self.buff[num_bytes_to_encrypt:]
```

Comment:
I deleted the space in front of ` self.buff` and made the function partly to deal with the long line (but also because I thought it was more readable)